kuma/javascript/src/payments/feedback-form.jsx

```diff
@@ -0,0 +1,106 @@
+// @flow
+import * as React from 'react';
+import { gettext, Interpolated } from '../l10n.js';
+import { getCookie } from '../utils.js';
+
+export const FEEDBACK_URL = '/payments/feedback';
+export const MIN_STRING_LENGTH = 10;
+
+const FeedbackForm = (): React.Node => {
+    const [feedback, setFeedback] = React.useState<string>('');
+    const [status, setStatus] = React.useState<'success' | 'idle'>('idle');
+    const [error, setError] = React.useState<React.Node | null>(null);
+
+    const handleChange = (event: SyntheticInputEvent<HTMLInputElement>) => {
+        const { value } = event.target;
+        setFeedback(value);
+    };
+
+    const handleSubmit = (event: SyntheticEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        const trimmedFeedback = feedback.trim();
+
+        // Feedback should be greater than 10 characters
+        if (trimmedFeedback.length < MIN_STRING_LENGTH) {
+            setError(
+                `To ensure more constructive feedback, a minimum of ${MIN_STRING_LENGTH} characters is required.`
+            );
+            return false;
+        }
+
+        fetch(FEEDBACK_URL, {
+            method: 'POST',
+            body: JSON.stringify({ feedback: trimmedFeedback }),
+            headers: {
+                'Content-Type': 'application/json',
+                'X-CSRFToken': getCookie('csrftoken')
+            }
+        })
+            .then(res => {
+                // Remove focus from button or input
+                if (document.activeElement) {
+                    document.activeElement.blur();
+                }
+
+                if (!res.ok) {
+                    throw new Error(
+                        'Request (POST) to /api/v1/payments/feedback failed'
+                    );
+                }
+                return res;
+            })
+            .then(() => {
+                // Clear form, show thank you message
+                setFeedback('');
+                setError('');
+                setStatus('success');
+            })
+            .catch(() => {
+                setError(
+                    <Interpolated
+                        id={gettext(
+                            "We're sorry, something went wrong. Please try again or send your feedback to <emailLink />."
+                        )}
+                        emailLink={
+                            <a
+                                target="_blank"
```

Comment:
Do you need to do that on a `mailto:` hyperlink??

Reply:
I think so! I use the browser version of Gmail, so when I remove the `_blank`, it opens up Gmail in the same tab, versus opening up Gmail in a new tab.